content/en/docs/reference/networking/virtual-ips.md

```diff
@@ -182,6 +162,24 @@ Especially, if kube-proxy's `sync_proxy_rules_duration_seconds` metric
 indicates an average time much larger than 1 second, then bumping up
 `minSyncPeriod` may make updates more efficient.
 
+##### Updating legacy `minSyncPeriod` configuration {#minimize-iptables-restore}
+
+{{< feature-state for_k8s_version="v1.28" state="stable" >}}
```

Comment:
With this heading, I'd probably omit the feature state. “legacy `minSyncPeriod` configuration” isn't a feature.

It's tricky, I know. But anyway I suggest:
```suggestion
```